### PR TITLE
Added documentation for usage/showback feature

### PR DESCRIPTION
**Pull request by adnaanm**

### CODE REVIEW

**Comment by ryanmillerdev** [comment]
Looping in @StephanieKim as well.

File: _pages/platform/usage-metrics.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+### Platform Usage Metrics
+
+Usage metrics of the platform can be viewed from the admin panel under `Platform Usage`. Given a customizable date range (which defaults to the past 7 days), users can view platform usage from three different perspectives: all usage, algorithms, and users. This is presented as a table in the UI, in which you can click on each row to get expanded information. The date range can be selected in preset intervals of the past seven, 14, or 30 days; or one of the past three months can be selected. You can also designate a more customized date range.
+
+
+# All Usage
+
+This tab shows information grouped by the username/algorithm tuple. That is, it shows which users called which algorithms and the number of times they did so (along with hardware, computer time, and other metrics).
```
Comment (stevielkim) [inline _pages/platform/usage-metrics.md]: Should this be "compute time" I think just a typo

File: _pages/platform/usage-metrics.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+layout: article
+title:  "Usage Mestrics"
```
Comment (stevielkim) [inline _pages/platform/usage-metrics.md]: Typo - I'm going to get some consensus on what we are calling this feature in product meeting today, I've talked a bit to other PM's last week and thinking something along the lines of "Usage Reports" but "Usage Metrics" works too - will let you know by EOD today. This doc looks great and thanks so much for doing it!

**Comment by adnaanm** [comment]
typos fixed, thanks a ton for the proof-read @StephanieKim!

File: _pages/platform/usage-metrics.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+show_related: true
+image:
+    teaser: /icons/algo.svg
+permalink: /platform/usage/
+redirect_from:
+  - /basics/usage/
+---
+
+
+### Platform Usage Metrics
```
Comment (bnlb) [inline _pages/platform/usage-metrics.md]: Let's update this to `## Platform Usage Metrics` so it ends up rendering as an `<h2>`

File: _pages/platform/usage-metrics.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- version: hash version of the exact algorithm called
+- owner: user who created and owns the algorithm
+- total calls: total number of calls that users of the platform made to this algorithm
+- total duration: total number of seconds the algorithm ran summed up across all users
+- total errors: total number of errors this algorithm returned summed up across all users
+- error duration: total number of seconds the algorithm ran calls in which it returned an error (summed up across all users)
+- hardware: whether the algorithm runs on a CPU or GPU instance
+
+
+# Users
```
Comment (bnlb) [inline _pages/platform/usage-metrics.md]: And then update each of these headers to `### All Usage`, `### Algorithms`,` ### Users` so they render as `<h3>`'s.

File: _pages/platform/usage-metrics.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+This tab shows usage information grouped at the user level, so you can see how many total calls each user made (across all algorithms).
+
+When you click on a row in the users table, you will see this complete set of columns:
+caller: the username of the user this row represents
+total calls: total number of calls the user made summed up across all algorithms
+total duration: total number of seconds the user’s calls spent computing summed up across all algorithms
+total errors: total number of errors returned to the user summed up across all algorithms
+error duration: total number of seconds computed by calls that returned errors to the user
```
Comment (bnlb) [inline _pages/platform/usage-metrics.md]: Screen Shot 2020-01-29 at 4 06 17 PM
This part's rendering a bit funky. Looks like it just needs the dashes.

**Comment by bnlb** [comment]
@adnaanm Thanks for adding this! If you'd like, I'm happy to deploy this during our prod deploy tomorrow.

**Comment by adnaanm** [comment]
Thanks for the comments @bnlb - yeah if it was possible to deploy tomorrow that'd be awesome!

**Review by bnlb — APPROVED** [review]
Looks great! Thanks for making those updates. 🌟

**Pull request merged**